src/utils/waitUntilTx.ts

```diff
@@ -0,0 +1,37 @@
+import { SyndicateClient } from "Client"
+
+interface WaitUntilTxOptions {
+  projectID: string
+  txID: string
+  maxAttempts?: number
+  every?: number
+}
+
+export async function waitUntilTx(client: SyndicateClient, {
+  projectID,
```

Comment:
Does a SyndicateClient not have the context of the project? I thought because a API key is tied to a project it would be locked in at client init?

Reply:
If we had an endpoint that returned the project ID we could make it work, but I can't figure out a way to get a project ID just given an API key